src/styles/index.scss

```diff
@@ -2,2 +1,57 @@
+@import 'vars';
+@import 'mixins';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+
+  @include flex-block;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &__day {
+    @include flex-block;
+    @include text-style;
+
+    width: $day-width;
+    height: $day-width;
+    background-color: $calendar-color;
+    color: black;
+    border: $day-border;
+    transition-duration: 500ms;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $pointer-calendar-color;
+      transform: translateY(-20px);
+    }
+
+    @for $numbers from 1 through 31 {
```

Comment:
Instead of hardcoding the number of days to 31, you can make your calendar more flexible by using a variable for the number of days.